components/ibm-components/watson/deploy/component.yaml

```diff
@@ -0,0 +1,33 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Watson Machine Learning - Deploy Model'
+description: |
+  Deploy stored model on Watson Machine Learning as a web service.
+inputs:
+  - {name: model_uid,       description: 'Required. UID for the Watson Machine Learning model'}
```

Comment:
Is this the UID for stored model?

Reply:
Yes, the Watson store component output will be passed to this field as model UID.